src/day8/re_frame/trace.cljs

```diff
@@ -234,8 +234,11 @@
             [:thead>tr
              [:th [:button.text-button
                    {:style {:cursor "pointer"}
-                    :on-click #(reset! trace-details {:showing (not (:showing @trace-details))})}
-                   (if (:showing @trace-details) "-" "+")]]
+                    :on-click (fn [e]
+                                (when (:showing @trace-detail-expansions)
+                                  (swap! trace-detail-expansions assoc :overrides {}))
```

Comment:
Same issue here with the `when` and two `swap!`s. This might be better? (haven't fully worked it through)

```clj
(swap! trace-detail-expansions
  (fn [t] (if (:showing t)
    {:showing false}
    {:showing true})))
```